### PR TITLE
attempted assessment

### PR DESCRIPTION
**Pull request by romeoyutig**
# Submitting Week 5 Assessment

### ✅ Completed Files

Mark the following as confirmation that the files in this repository are complete.

- [x] JavaScript Coding Challenges
- [x] Ruby Coding Challenges
- [x] Interview Questions

### 📝 Feedback

If you would like specific feedback on a particular problem please leave details here:
Did not complete looking ahead section of interview questions.

### CODE REVIEW

File: interview-questions.md [hunk 1]
```diff
@@ -8,33 +8,33 @@ Then, research the question to expand on your answer. Even if you feel you have
...
 
-Your answer:
+Your answer: A gem is a ruby dependency file. We use it mainly for testing. There are many ruby gems. 
 
-Researched answer:
+Researched answer: A gem is a snippet of code functionality that is managed by a packet manager. It allows you to download, and use ruby software packages on your system. They add large amounts of functionality to your software.
 
 3. What is a relational database? Are there other kinds of databases?
 
 Your answer:
```
Comment (TsunamiCode5) [inline interview-questions.md]: I think you missed this question, can you please go back and complete this so I can review it

File: interview-questions.md [hunk 2]
```diff
@@ -8,33 +8,33 @@ Then, research the question to expand on your answer. Even if you feel you have
...
-Your answer:
+Your answer: A primary key is the key that can be references in other databases. You can use this to connect databases to each other, cutting out on unecessary rows and cutting down on null values.
 
-Researched answer:
+Researched answer: A primary key is the key that can be references in other databases. You can use this to connect databases to each other, cutting out on unecessary rows and cutting down on null values. Rails will assign a primary key whenever a new table is created.
 
 5. What are the HTTP verbs? What is each verb's corresponding CRUD action?
 
-Your answer:
+Your answer:
```
Comment (TsunamiCode5) [inline interview-questions.md]: I think you missed this one too

**Review by TsunamiCode5 — CHANGES_REQUESTED** [review]
Before I can review your assessment, the looking ahead terms need to be answered, please. Also, I think you missed Interview Questions #3 and #5. Please answer these and slack me when you have added, committed, and pushed to this branch again. The PR will update itself when you push your answers back to the branch.

File: ruby-challenges.rb [hunk 3]
```diff
@@ -10,18 +10,61 @@
...
+# create a method called letter_checker
+# input: array of words, and a single letter
+# expected output: an array of all the words containing that particular letter.
+# process: create a method called letter_checker with the parameters 'array', and 'string'
+# use the select method on the parameter array
+# print the select method to check if each array in the element includes the string provides in the parameter 
+# call the letter_checker method with the test variables provided included in the argument
+
+def letter_checker(array, string)
+  p array.select { |array| array.include?(string)}
```
Comment (TsunamiCode5) [inline ruby-challenges.rb:24]: Nice!

File: code-challenges.test.js [hunk 4]
```diff
@@ -49,4 +122,36 @@ const hand3 = [5, 5, 5, 5, 4]
...
+
+// ReferenceError: hand_Checker is not defined
+// good test, hand_Checker is not yet declared
+
+
 // b) Create the function that makes the test pass.
+
+const hand_Checker = (array) => {
+  let new_Array = array.sort((a,b) => (a - b)) 
+    if(new_Array[0] === new_Array[1] && new_Array[2] === new_Array[3] && new_Array[3] === new_Array[4])
```
Comment (TsunamiCode5) [inline code-challenges.test.js:152]: Awesome!

File: code-challenges.test.js [hunk 5]
```diff
@@ -14,15 +14,61 @@
...
+  "e" : 3 ,
+  "E" : 3 ,
+  "i" : 1 ,
+  "I" : 1 ,
+  "o" : 0 ,
+  "O" : 0
+}
+
+const translator = (str) => {
+  return str.replace(/[aeio]/gi, (str) =>
```
Comment (TsunamiCode5) [inline code-challenges.test.js:60]: Really great job on this one.

File: interview-questions.md (hunk 1 above)
Comment (romeoyutig) [reply interview-questions.md]: Hey Nicole, I went back and completed this answer, thanks.

File: interview-questions.md (hunk 2 above)
Comment (romeoyutig) [reply interview-questions.md]: I went back and completed this as well.

**Review by TsunamiCode5 — APPROVED** [review]

**Pull request merged**